package.json

```diff
@@ -17,11 +19,12 @@
     "xrpl": "file:packages/xrpl"
   },
   "devDependencies": {
+    "@jest/types": "^29.3.1",
     "@types/chai": "^4.2.21",
     "@types/create-hash": "^1.2.2",
+    "@types/jest": "^29.2.2",
```

Comment:
Do we need both `@jest/types` and `@types/jest`? (My quick google found this thread on the topic: https://github.com/facebook/jest/issues/9972)

Reply:
No we do not